common/app/views/fragments/atoms/storyquestions.scala.html

```diff
@@ -0,0 +1,33 @@
+@(storyquestions: model.content.StoryQuestionsAtom, isAmp: Boolean)(implicit request: RequestHeader)
+
+@if(!isAmp) {
+    <div class="submeta user__question">
+    <span class="js-storyquestion-atom-id is-hidden" id="@storyquestions.id"></span>
+
+    <h1 class="user__question-title">Finished this article but want to know more? <span class="user__question-title--secondary">Let us know what questions you want answered.</span></h1>
```

Comment:
IMHO this should never be a `<h1>`, but only the article-headline, but I'm not sure how we deal with atoms and headline structure to be honest.